### PR TITLE
[ADD] contract_recurring_plans

### PR DESCRIPTION
**Pull request by pedrobaeza**
# 
# Analytic plans for contract recurring invoices

This module allows to include an analytic plan on each recurring invoice line,
so if you set it, this will be taken to be transferred to the invoice instead
of the analytic account of the contract.
# Usage

On the contract view, select or create an analytic distribution for the
invoice recurring lines that you want.

### CODE REVIEW

File: contract_recurring_plans/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Serv. Tecnol. Avanzados - Pedro M. Baeza
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    'name': 'Analytic plans on contracts recurring invoices',
+    'version': '1.0',
```
Comment (dreispt) [inline contract_recurring_plans/__openerp__.py]: 8.0.1.0.0

File: contract_recurring_plans/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    'category': 'Other',
```
Comment (dreispt) [inline contract_recurring_plans/__openerp__.py]: "Contract"?

File: contract_recurring_plans/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    'author': 'Serv. Tecnolog. Avanzados - Pedro M. Baeza',
```
Comment (dreispt) [inline contract_recurring_plans/__openerp__.py]: Add OCA

**Comment by dreispt** [comment]
Other than some minor notes on the manifest, it's :+1:

**Comment by sergio-teruel** [comment]
I have tested it on runbot and for me :+1:

**Comment by pedrobaeza** [comment]
@dreispt Changes done

**Comment by dreispt** [comment]
Needs a third review or 5 days.

**Comment by pedrobaeza** [comment]
Merging as this is already approved.

**Pull request merged**